api/bin/chainflip-broker-api/Cargo.toml

```diff
@@ -34,6 +35,7 @@ jsonrpsee = { workspace = true, features = ["full"] }
 serde = { workspace = true, default-features = true, features = ["derive"] }
 sp-core = { workspace = true }
 sp-rpc = { workspace = true }
+sc-rpc = { workspace = true, default-features = true }
```

Comment:
Duplicate 👀 

FWIW I think sp-core should also have `default-features=true`

Reply:
Unfortunately one is `sP-` and the other `sC-`... The latter was required for the `sc_rpc::utils::pipe_from_stream()` utility.

I am not sure about the default-features though, I copied it from the `custom_rpc` lib. In fact it compiles if I remove the `default-features=true` for `sc-rpc`.

Should I enable default-features for all three of them anyways?

Reply:
Yes we should enable it for all three - it might not be a problem now, but this kind of thing tends to cause problems unexpectedly.